### DISCUSSION TITLE
Create audit for a pivot table change

### DISCUSSION THREAD

**Discussion by newave-ws** [owen-it/laravel-auditing#927] [Q&A] [answered]
I have client model that have a pivot called patients which is connecting User model with the following code:
(this is the code that is in the client model)
```php
public function patients() {
    return $this->belongsToMany('App\Models\User' , 'client_patient' , 'client_id', 'patient_id')
        ->withPivot(
            'name', 'first_name', 'last_name', 'email', 'social_id', 'phone', 'age', 'dateOfBirth',
            'address', 'city', 'gender', 'hmo', 'insurance', 'can_edit', 'verified', 'archive'
        )
        ->withTimestamps();
}
```
I want to know how can I add an audit on the pivot table each time that is been created or updated?
Help will be very appreciated

**Comment by parallels999** [comment, marked as answer]
Did you try [audit-custom.html#example-related-attributes](https://laravel-auditing.com/guide/audit-custom.html#example-related-attributes)?

Also take a look at #921, https://github.com/owen-it/laravel-auditing/pull/684/commits/92078685671e56a48a065ed562e4c91ac2f48759, https://github.com/owen-it/laravel-auditing/pull/684/commits/fa10bfeec772b7d207a580710eb5530e2c9882e4

**Reply by newave-ws** [reply]
Do you know how to implement this on a pivot table? because i don't really understand the two options that you suggested

**Reply by parallels999** [reply]
>Did you try [audit-custom.html#example-related-attributes](https://laravel-auditing.com/guide/audit-custom.html#example-related-attributes)?

Just read the documentation, it seems it is already implemented, so just give it a try and tell us how it went.

**Reply by willpower232** [reply]
@newave-ws did you try the above with the latest version?

**Reply by newave-ws** [reply]
@willpower232 do you mean the latest version of the library?

**Reply by newave-ws** [reply]
@willpower232 i don't really understand how to implement that and how it's adding audits to a pivot table or getting the audits of the pivot table.

**Reply by newave-ws** [reply]
@erikn69 i still got the problem, can you help me?

**Reply by erikn69** [reply]
> i don't really understand how to implement

Did you read first answer? In my opinion the documentation is very clear.
Show to us, how do you tried? maybe you wrote something wrong
